holoviews/element/raster.py

```diff
@@ -259,6 +260,18 @@ def __init__(self, data, bounds=None, extents=None, xdensity=None, ydensity=None
                 raise ValueError("Input array has shape %r but %d value dimensions defined"
                                  % (self.shape, len(self.vdims)))
 
+
+    def __setstate__(self, state):
+        """
+        Ensures old-style unpickled Image types without an interface
+        use the ImageInterface.
+        """
```

Comment:
Just one quick comment: might be worth adding the word 'deprecation' so when we grep for it we find it (e.g if we no longer want to support this in 2.0)